db/LMS.sql

```diff
@@ -0,0 +1,16 @@
+DROP DATABASE IF EXISTS lms;
+CREATE DATABASE lms;
+
+\connect lms
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE USERS(
+    id uuid UNIQUE DEFAULT uuid_generate_v4(),
+    email VARCHAR(128) NOT NULL UNIQUE PRIMARY KEY,
+    password VARCHAR(128) NOT NULL,
+    role VARCHAR(10) NOT NULL,
+    token VARCHAR(50) NOT NULL,
+    isVerified BOOLEAN
```

Comment:
If the created date and last updated fields are not automatically added to you table, kindly include.